.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
   },
   extends: [
     "react-app",
+    "react-app/jest",
```

Comment:
🗒️ CRA includes this extend now. I wonder if that includes `plugin:jest/recommended`